corehq/apps/export/views.py

```diff
@@ -125,6 +125,17 @@
 from six.moves import map
 
 
+# Form used for rendering filters
+def _get_filter_form_class(form_or_case, sms_export=False):
```

Comment:
What's the advantage of this approach?  Are you hoping to eventually merge all the download views?

Reply:
I think this comment is addressed by the rearranging done in 177fb0b3af3fef1f83ca8e145c91ec3a4329f275, but let me know if it isn't.